doc/config/haproxy.cfg

```diff
@@ -36,4 +42,7 @@ backend test_backend
 
 backend coraza-spoa
     mode tcp
-    server s1 127.0.0.1:9000
+    balance roundrobin
```

Comment:
There is the only one server, so it's not necessary. Also it's not related to WAF config. Suggest to remove.

Reply:
My comment is for line 45 `balance roundrobin`.

Reply:
oh 🙈 its been a too long week it seems, reading comprehension is failing